ports/raspberrypi/boards/challenger_rp2040_wifi/pins.c

```diff
@@ -42,8 +42,8 @@ STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_TX), MP_ROM_PTR(&pin_GPIO16) },
     { MP_ROM_QSTR(MP_QSTR_GP16), MP_ROM_PTR(&pin_GPIO16) },
 
-    { MP_ROM_QSTR(MP_QSTR_D0), MP_ROM_PTR(&pin_GPIO17) },
-    { MP_ROM_QSTR(MP_QSTR_TX), MP_ROM_PTR(&pin_GPIO17) },
+    { MP_ROM_QSTR(MP_QSTR_D1), MP_ROM_PTR(&pin_GPIO17) },
```

Comment:
I checked the corrections against here, but why is there a `D1`, which is not on the silk nor the diagram?
![image](https://user-images.githubusercontent.com/2847802/134775747-09dce519-b3ad-443a-95c3-e652365257e7.png)

Reply:
Good call. It's a remnant from arduino boards where the uart pins were referred to as d0 and d1 when used as gpio pins. I will update the pin diagrams to reflect this, for all our boards.